java-symbolic-execution/src/main/java/org/sonar/java/se/checks/NonNullSetToNullCheck.java

```diff
@@ -166,7 +169,9 @@ private void checkVariable(CheckerContext context, MethodTree tree, final Symbol
 
   private static boolean isJavaxValidationConstraint(Symbol symbol) {
```

Comment:
Maybe it's worth renaming the method  to reflect that it also checks for jakarta constraints, something like `isJavaxOrJakartaValidationConstraint`